src/openrct2/interface/Viewport.cpp

```diff
@@ -1,4 +1,4 @@
-﻿/*****************************************************************************
+/*****************************************************************************
```

Comment:
*cough*

Reply:
Weirdly, it seems to always happen in this file.